interface/wx/event.h

```diff
@@ -1741,10 +1741,17 @@ class wxJoystickEvent : public wxEvent
     /**
         Returns the identifier of the button changing state.
 
-        This is a @c wxJOY_BUTTONn identifier, where @c n is one of 1, 2, 3, 4.
+        This was once a @c wxJOY_BUTTONn identifier, where @c n is one of 1, 2, 3, 4,
+        and whose values were 1 << n. To support more than four buttons, the return value
+        is now defined as 1 << n.
     */
     int GetButtonChange() const;
 
+    /**
+        Returns the 0-indexed ordinal of the button changing state @since 3.1.2.
```

Comment:
You need a full stop before `@since`, as it won't be rendered correctly otherwise, I think. I.e.

```suggestion
        Returns the 0-indexed ordinal of the button changing state.
        
        @since 3.1.2.
```